code/citadel/dogborgstuff.dm

```diff
@@ -389,18 +414,42 @@
 		if(patient) return //If you try to eat two people at once, you can only eat one.
 
 		else //If you don't have someone in you, proceed.
+			if(!isslimeperson(target) && ("toxin" in injection_chems))
```

Comment:
isslimeperson() does not take into account jellypeople, a species that also takes inverted toxin damage. It'd be more efficient to check for the TOXINLOVER flag in the target species' traits instead.

Reply:
Yes it does.